apps/shell/tash_main.c

```diff
@@ -121,12 +121,12 @@ char *tash_read_input_line(int fd)
 	int nbytes;
 	int char_idx;
 	bool is_tab_pressed = false;
-#if TASH_MAX_STORE > 0
+	int prev_cmd_len = 0;
```

Comment:
Please use tab-spaces at the beginning of the line of code instead of multiple single spaces